x-pack/plugins/alerts/server/lib/license_state.test.ts

```diff
@@ -27,8 +29,8 @@ describe('license_state', () => {
     it('check application link should be disabled', () => {
       const licensing = licensingMock.createSetup();
       const licenseState = new LicenseState(licensing.license$);
-      const alertingLicenseInfo = licenseState.checkLicense(getRawLicense());
-      expect(alertingLicenseInfo.enableAppLink).to.be(false);
+      const actionsLicenseInfo = licenseState.checkLicense(getRawLicense());
```

Comment:
Looks like there's a few variables in this file referencing actions terminology when it should be alerts.